src/zm_videostore.cpp

```diff
@@ -952,11 +949,15 @@ int VideoStore::writePacket(ZMPacket *ipkt) {
   if ( ipkt->packet.stream_index == video_in_stream_index ) {
     return writeVideoFramePacket(ipkt);
   } else if ( ipkt->packet.stream_index == audio_in_stream_index ) {
-    return writeAudioFramePacket(ipkt);
+    if ( monitor->RecordAudio() )
```

Comment:
So... I don't like doing extra function calls and if's on super frequently called code.  If we aren't recording audio, then audio_in_stream_index = -1.  Also we have a check in Monitor::Capture for this as well.  So it SHOULD never happen that we are given a packet matching audio_in_stream_index.

Reply:
Apologies, theses changes shouldn't have landed in this PR. I removed them. They are very much WIP.